scripts/test-unit-tests

```diff
@@ -64,9 +64,22 @@ function analyzeBuild() {
         -workspace Branch-TestBed.xcworkspace \
         -scheme Branch-TestBed \
         -destination name="iPhone 7",OS=latest \
-            clean analyze -quiet
+            analyze -quiet
```

Comment:
One of these days I'll give you one Fastfile that replaces this whole folder. :)